test/tests/ver-1kd/comparison_ver-1kd.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pandas as pd
+import os
+from matplotlib import gridspec
+import matplotlib.pyplot as plt
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+script_folder = os.path.dirname(__file__)
+os.chdir(script_folder)
+
+# Load experimental data
+if "/TMAP8/doc/" in script_folder:     # if in documentation folder
```

Comment:
```suggestion
if "/tmap8/doc/" in script_folder.lower():     # if in documentation folder
```
We now use this approach, which enable people to have their folder named either TMAP8 or tmap8.